rdt/transformers/numerical.py

```diff
@@ -1,16 +1,58 @@
 """Transformers for numerical data."""
-
 import copy
 
 import numpy as np
 import pandas as pd
 import scipy
-from copulas import EPSILON, univariate
 
 from rdt.transformers.base import BaseTransformer
 from rdt.transformers.null import NullTransformer
 
 
+def _get_distributions():
+    from copulas import univariate  # pylint: disable=import-outside-toplevel
```

Comment:
Let's wrap this in a `try/except` block and provide a user friendly message with instructions about what to do.

Similar to this: https://github.com/sdv-dev/SDV/blob/8f3c787dc87ff2b85fe1ac392e0decf18ce134ad/sdv/tabular/ctgan.py#L93-L102

In this case, let's recommend installing `rdt[copulas]`